Makefile

```diff
@@ -171,6 +171,9 @@ pii_clean:
 isort: ## call isort on packages/files that are checked in quality tests
 	isort --skip migrations --recursive tests test_utils enterprise enterprise_learner_portal consent integrated_channels manage.py setup.py
 
+%-shell: ## Run a shell, as root, on the specified service container
```

Comment:
because there's no actual application running per se, "app" feels a bit misleading to me. maybe something more like `make test-env-shell` or `make test-shell` or something. what do you think?

Reply:
I had been thinking about that a bit too. I wasn't sure if it was more useful to maintain `make app-shell` since that was familiar to users, or ultimately more confusing since users may expect it to be a real running instance of the project (web service, workers, database, etc).

I like `make test-shell` for now.

Reply:
+1, while the familiarity of `app-shell` is a plus, it is misleading. Only added feedback here is to maybe add `make test-shell` known somewhere on the README.

Reply:
I'll add notes to the `local development` parts of `doc/` 👍